pom.xml

```diff
@@ -100,6 +100,9 @@
         <hibernate-orm.plugin.version>6.2.13.Final</hibernate-orm.plugin.version>
         <hibernate.c3p0.version>6.2.13.Final</hibernate.c3p0.version>
         <infinispan.version>15.0.5.Final</infinispan.version>
+        <!-- Remove the explicit protostream dependency once Infinispan has caught up.
+        See https://github.com/keycloak/keycloak/issues/31032 that is tracking this -->
+        <protostream.version>5.0.7.Final</protostream.version>
```

Comment:
we cannot remove this since we need the version for the plugin.

Reply:
except if you want to re-introduce `infinispan.maven-plugins.version`.

Reply:
Argh good point @pruivo, I didn't connect the dots. I'll remove this comment and close the issue.